### PR TITLE
Add landing message.#1

### PR DESCRIPTION
**Pull request by FiqDin**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
   <title>Welcome to Potionfy</title>
 </head>
 <body>
-  
+    <h1>Welcome ba to Portionfy! Enjoy your stay.</h1>
```
Comment (FiqrieAiddin) [inline index.html]: Could you remove the 'Enjoy your stay.' string and makes the 'Welcome ba to Potionfy!' string red?

**Review by FiqDin — COMMENTED** [review]
sure can!

**Review by FiqrieAiddin — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by FiqrieAiddin — APPROVED** [review]
Well done Fiqrie!

**Pull request closed without merging**